crates/libwebrtc-sys/build.rs

```diff
@@ -70,13 +79,31 @@ fn main() -> anyhow::Result<()> {
 /// Downloads and unpacks compiled `libwebrtc` library.
 fn download_libwebrtc() -> anyhow::Result<()> {
     let mut libwebrtc_url = env::var("LIBWEBRTC_URL")?;
```

Comment:
I guess we can use `.unwrap_or()` here to default into the version we intend to use.

Additionally, we need to specify and verify SHA-256 checksums here.